app/lib/test_applications.rb

```diff
@@ -106,7 +106,7 @@ def create_application_to_courses(
     elsif carry_over
       create_application_to_courses(
         recruitment_cycle_year: recruitment_cycle_year - 1,
-        courses: courses,
+        courses: Course.open_on_apply.in_cycle(recruitment_cycle_year - 1),
```

Comment:
aren't the courses passed into this method scoped?

Reply:
Yes I thought about this - glad you brought it up!

This is generating an application from "last year" from which this new test application is being carried over. In real life I think those courses could be anything, so that's what I've gone with here.

We only use the `carry_over` option in the generate test data task, so it won't affect API test data